boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090.rst

```diff
@@ -0,0 +1,207 @@
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF91 DK website`_ and `Nordic Semiconductor Documentation library`_
+for a complete list of nRF9160 Development Kit board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.2
+* LED2 (green) = P0.3
+* LED3 (green) = P0.4
+* LED4 (green) = P0.5
+
+Push buttons and Switches
+-------------------------
+
+* BUTTON1 = P0.6
+* BUTTON2 = P0.7
+* SWITCH1 = P0.8
+* SWITCH2 = P0.9
+* BOOT = SW5 = boot/reset
+
+Security components
+===================
+
+- Implementation Defined Attribution Unit (`IDAU`_).  The IDAU is implemented
+  with the System Protection Unit and is used to define secure and non-secure
+  memory maps.  By default, all of the memory space  (Flash, SRAM, and
+  peripheral address space) is defined to be secure accessible only.
+- Secure boot.
+
+
+Programming and Debugging
+*************************
+
+nrf9160_pca10090 supports the Armv8m Security Extension, and by default boots
+in the Secure state.
+
+Building Secure/Non-Secure Zephyr applications
+==============================================
+
+The process requires the following steps:
+
+1. Build the Secure Zephyr application using ``-DBOARD=nrf9160_pca10090`` and
+``CONFIG_TRUSTED_EXECUTION_SECURE=y`` in the the application project configuration file.
+2. Build the Non-Secure Zephyr application using ``-DBOARD=nrf9160_pca10090ns``.
+3. Merge the two binaries together.
+
+When building a Secure/Non-Secure application, the Secure application will
+have to set the IDAU (SPU) configuration to allow Non-Secure access to all
+CPU reources utilized by the Non-Secure application firmware. SPU
+configuration shall take place before jumping to the Non-Secure application.
+
+Building a Secure only application
+==================================
+
+Build the Zephyr app in the usual way, using ``-DBOARD=nrf9160_pca10090``.
```

Comment:
Do folks know what "the usual way" is?  Maybe change to:
```
Build the Zephyr app in the usual way (see :ref:`build_an_application`
and :ref:`application_run`), using ``-DBOARD=nrf9160_pca10090``.
```

Reply:
Fine, I incorporated your suggestion.